src/contrib/mlir/runtime/cpu/cpu_runtime.cpp

```diff
@@ -129,39 +129,37 @@ void MLIRCPURuntime::cleanup()
     // Free void double pointer arguments without freeing external tensor data.
     for (auto* arg : m_invokeArgs)
     {
-        auto* memRefArg = *(reinterpret_cast<mlir::StaticFloatMemRef**>(arg));
+        auto* memRefArg = *(reinterpret_cast<StaticMemRef**>(arg));
```

Comment:
How is MLIR replacing `StaticFloatMemRef`?

Reply:
I haven't seen any replacement. JITRunner has its own struct as well. If you know of any, please point me to it.